agent/go.mod

```diff
@@ -0,0 +1,114 @@
+module github.com/pulumi/pulumi-kubernetes-operator/agent
+
+go 1.22.5
+
+require (
+	github.com/fluxcd/pkg/http/fetch v0.11.0
+	github.com/gogo/protobuf v1.3.2
+	github.com/grpc-ecosystem/go-grpc-middleware v1.4.0
+	github.com/onsi/gomega v1.32.0
+	github.com/pulumi/pulumi/sdk/v3 v3.127.1-0.20240801092425-22d28187db0a
+	github.com/spf13/cobra v1.8.1
+	go.uber.org/zap v1.26.0
+	google.golang.org/grpc v1.65.0
+	google.golang.org/protobuf v1.34.2
+	k8s.io/utils v0.0.0-20230726121419-3b25d923346b
+)
+
+// Replace digest lib to master to gather access to BLAKE3.
+// xref: https://github.com/opencontainers/go-digest/pull/66
+replace github.com/opencontainers/go-digest => github.com/opencontainers/go-digest v1.0.1-0.20220411205349-bde1400a84be
```

Comment:
Any background on this? I'm not familiar with it, just curious what it gets us.

Reply:
Thanks for the reminder, I added a link to where this originated (fluxcd):
https://github.com/fluxcd/pkg/blob/968f1907a8d27b8a13ee40fab88ae7dbdb553539/http/fetch/go.mod#L10-L12